### ISSUE TITLE
Action Sheet's focus issue

### ISSUE THREAD

**Issue by anatolybolshakov** [ionic-team/ionic#15581]
# Bug Report

With ionic Action Sheet I've found some focus issue, which can confuse people using some Accessibility technologies (like TalkBack in Android).  Here's the Stackblitz for trying it:
https://stackblitz.com/edit/ionic-axoli9?file=app

**Steps to Reproduce**
1. Click on the "Open action sheet" button
2. See that focus is not going to the action sheet, e.g. to the "Option 1" button (blue focus border is not visible). This can be obstacle for users who navigates via Accessibility tools

You can also try to reproduce it using Edge and Narrator screen reader (or TalkBack on Android/VoiceOver on iOS). All of them mark current focus with a border and it's on appearing in Action Sheet after opening it. Maybe calling .focus() for the first Action Sheet option element on panel's opening could solve this?

Thanks!

**Comment by QiaoyuanMaxDeng** [comment]
We had the same issue. So far based on my research ionic v3 does not support accessibility and v4 might have some support but might not be fully support it. 

Based on our testing, popup focus works fine but action sheet does not be focused at all. We are using customized JS to manually focus on first item.
